docs/se/metrics/micrometer.adoc

```diff
@@ -23,14 +23,45 @@
 
 include::{rootdir}/includes/se.adoc[]
 
+== Contents
+- <<Overview, Overview>>
+- <<Maven Coordinates, Maven Coordinates>>
+- <<Usage, Usage>>
+- <<API, API>>
+- <<Configuration, Configuration>>
+- <<Examples, Examples>>
+- <<Additional Information, Additional Information>>
+
 include::{rootdir}/includes/metrics/micrometer-shared.adoc[tags=intro]
 include::{rootdir}/includes/metrics/micrometer-shared.adoc[tags=prereq]
 
-== Using Micrometer in Your Application
-You need to make two types of changes to your application to use Helidon SE integration with Micrometer:
+== Usage
```

Comment:
Should there be some include here for usage? There is no text under the H2. Is that intentional?

Reply:
There was not missing include, but upon (re)reading the Oracle doc guidelines document you pointed me to I realize it's discouraged to have a header followed immediately by a subheader. I've added some brief intro text to both the SE and MP versions of this file.